aliyun/cr/api.go

```diff
@@ -0,0 +1,101 @@
+package cr
```

Comment:
cr?

Reply:
aliyun refer Container Registry as `cr` that's why

you prefer `containerregistry` ?

Reply:
I think idiomatic golang would be `registry`. Single words for package names and all that.